docs/diagrams/EditParticipantCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+ hide footbox
+
+ participant ":Main" as main
+ participant ":EditParticipantCommand" as editCommand
+ participant ":EventList" as eventList
+ participant ":Event" as event
+
+ main -> editCommand : execute()
+ activate editCommand
+
+ editCommand -> eventList : editParticipant(participantName, participantNumber, participantEmail, eventName)
+ activate eventList
+
+ loop for each event in eventList
+     eventList -> event : getEventName()
+     activate event
+     event --> eventList : eventName
+     deactivate event
+
+     alt eventName matches
+         eventList -> event : updateParticipant(participantName, participantNumber, participantEmail)
+         activate event
+         event --> eventList : updated
+         deactivate event
+         eventList --> editCommand : true
```

Comment:
For the return arrows here, I think you can label them as ":boolean", rather than "true" or "false"?